newsfragments/2989.feature.rst

```diff
@@ -0,0 +1 @@
+:class:`trio.testing.RaisesGroup` can now catch an unwrapped exception with ``unwrapped=True``, similar to ``except*``.
```

Comment:
It should also mention `flatten_subgroups`.  Please also update PR description with this info (including deprecation of `strict`.